src/main/groovy/jenkins/automation/builders/PipelineJobBuilder.groovy

```diff
@@ -32,7 +32,9 @@ class PipelineJobBuilder {
     Job build(DslFactory factory) {
         factory.pipelineJob(name) {
             it.description this.description
-            CommonUtils.addDefaults(delegate)
+            logRotator {
```

Comment:
Why was this line changed?

Reply:
This resolves issue #140 (basically parts of the addDefaults method were deprecated in the new version of the job DSL plugin

Reply:
https://github.com/cfpb/jenkins-automation/blob/master/src/main/groovy/jenkins/automation/utils/CommonUtils.groovy#L14  We should fix the method. This is mostly for uniformity, so we can change that logRotator value if we need to.

Reply:
this was the change requested by @marcesher as the common utils method didn't have any additional non deprecated features beyond the logRotator